manifests/profile/www_lib/vhosts/apps_lib.pp

```diff
@@ -231,7 +231,7 @@
     ssl_proxy_machine_cert        => $client_cert,
 
     custom_fragment               => @(EOT)
-      ProxyPassReverse / https://sali1.lib.umich.edu:8443/
+      ProxyPassReverse / https://sali.lib.umich.edu:8443/
     | EOT
```

Comment:
We shouldn't need any of lines 228-235, given we are not proxying to an app anymore.

Reply:
Ah, ok I misunderstood.  I Thought Albert was asking that we proxy back to sali.lib now that its hosted in AWS.